### PR TITLE
Restructure vamp

### LINKED ISSUE #76: notebook 7
double-well example:
-----
- vamp is used without any arguments. That is bad practice because at least the user will want to select a lagtime. 
- For this example the default lag time (10) seems too large, because the second singular value only appears to have a contribution of 0.08.
- There is no explanation about the VAMP values, e.g. why the VAMP score is 1 for the bad direction and what this means.

alanine dipeptide
-----
- the RMSD is really pointless, remove that. Use something else instead, such as Cartesian coordinates after aligning to a reference structure. That should still be worse than using the dihedrals.
- I don't think a direct application of VAMP to dihedral makes sense, because dihedrals are periodic but VAMP is linear. Use cos/sin of dihedrals.
- RMSD and dihedral VAMP scores and MSM analyses are in a confusing order, this is hard to follow, needs to be cleaned up.

**I suggest this notebook to be reviewed by @fabian-paul **

### PR DESCRIPTION
**Pull request by marscher**
Integrated the VAMP feature (scoring and dimension reduction) in notebooks 1 and 2. In the first one, the score is just being computed for each feature, but there is not yet a systematic comparison  etc.
This is added in the second notebook, in which the method is introduced along with PCA and TICA. The pentapeptide is used as a showcase on how to select a feature based on the VAMP score.
The di-alanine model is used for exercises.

Fixes #76, #81

### CODE REVIEW

**Comment by marscher** [comment]
Another goodie added in this PR is that we will store the output of the executed cells in new notebook files, transform these to HTML and store them as build artififacts within circle ci. However the service has degraded performance right now, this is why the build is not succeeding.

**Comment by marscher** [comment]
ready for review. You should be able to look at the output at the artifacts section of CircleCI, once tests have passed.

**Comment by thempel** [comment]
Maybe, since you are working on it, can you already delete the pentapeptide from notebook 01? (If that's desired.)

File: notebooks/02-dimension-reduction-and-discretisation.ipynb [hunk 1]
```diff
@@ -568,96 +626,124 @@
...
    "outputs": [],
    "source": [
-    "pdb = mdshare.fetch('pentapeptide-impl-solv.pdb', working_directory='data')\n",
-    "files = mdshare.fetch('pentapeptide-*-500ns-impl-solv.xtc', working_directory='data')\n",
-    "\n",
     "feat = pyemma.coordinates.featurizer(pdb)\n",
-    "feat.add_backbone_torsions(cossin=True)\n",
-    "feat.add_sidechain_torsions(which='chi1', cossin=True)\n",
-    "\n",
+    "feat. #FIXME\n",
```
Comment (thempel) [inline notebooks/02-dimension-reduction-and-discretisation.ipynb]: Solution cell missing.

Comment (thempel) [reply notebooks/02-dimension-reduction-and-discretisation.ipynb]: Sorry, not missing, but somehow a couple of cells further. What about the cells in between? They seem displaced...

File: notebooks/02-dimension-reduction-and-discretisation.ipynb [hunk 2]
```diff
@@ -755,15 +835,14 @@
...
-    "Have you noticed that increasing the lag time\n",
-    "1. leads to a rotation of the projection and\n",
-    "2. reduces the number of ICs to explain $95\\%$ (default) of the kinetic variance?\n",
+    "TICA, by default, uses a lag time of $10$ steps and a kinetic variance cutoff of $95\\%$ to determine the number of ICs. We observe that this projection does resolve some metastability in both ICs.\n",
     "\n",
-    "Note that, while we can get lower and lower dimensional subspaces with increased lag times, we also loose information from the faster processes.\n",
+    "Whether these projections are suitable for building Markov state models, though, remains to be seen in later tests.\n",
     "\n",
-    "How to choose the optimal lag time for a TICA projection often is a hard problem, but we will learn systematic approaches in later notebooks.\n",
+    "### Exercise 6: PCA parameters\n",
```
Comment (thempel) [inline notebooks/02-dimension-reduction-and-discretisation.ipynb]: Maybe, we can remove the PCA parameters exercise. The notebook is quite long already and people might be more interested in learning TICA. Tagging @cwehmeyer

**Comment by thempel** [comment]
Requesting changes according to #85:
NB02: lines 515 and 883 `cluster_kmeans(pca_all...)`
NB02: lines 573 and 940 `cluster_kmeans(tica_all...)`

File: notebooks/01-data-io-and-featurisation.ipynb [hunk 3]
```diff
@@ -456,12 +526,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import nglview, mdtraj\n",
```
Comment (cwehmeyer) [inline notebooks/01-data-io-and-featurisation.ipynb]: redundant imports (already done for the ala2 case)

Comment (marscher) [reply notebooks/01-data-io-and-featurisation.ipynb]: if we assume strictly linear execution order of the test systems, you are right. I thought we agreed upon making them more independent. I can also add these imports to the top to be clear.

Comment (cwehmeyer) [reply notebooks/01-data-io-and-featurisation.ipynb]: > I can also add these imports to the top to be clear.

That is a good idea.

**Comment by marscher** [comment]
Thanks for the review guys, I've added the requested changes.
